Python/sqlmlutils/sqlpythonexecutor.py

```diff
@@ -17,8 +17,9 @@
 
 class SQLPythonExecutor:
 
-    def __init__(self, connection_info: ConnectionInfo):
+    def __init__(self, connection_info: ConnectionInfo, language_name: str = "Python"):

```

Comment:
In python, Isn't there any documentation for this new language_name parameter that we should update ? similar to the .Rd files in the R/man

Or just mention as a comment in an externally visible sqlmlutils API that takes language_name parameter..